app/javascript/stylesheets/home.css

```diff
@@ -61,25 +61,22 @@
 }
 
 .podcast {
+  background: url('../../assets/images/waveform.svg');
+  background-repeat: repeat-x;
+  background-position: center;
+  background-size: auto 85%;
```

Comment:
This code sets the background of the `podcast` div to be that waveform background, repeating across the x axis and spanning 85% of the height.